spec/services/complete_moab_handler_check_exist_spec.rb

```diff
@@ -379,18 +380,22 @@
 
       it 'calls CompleteMoab.save! (but not PreservedObject.save!) if the existing record is NOT altered' do
         druid = 'zy987xw6543'
-        po = create(:preserved_object, druid: druid)
-        cm = create(:complete_moab, preserved_object: po)
-        allow(PreservedObject).to receive(:find_by!).with(druid: druid).and_return(po)
-        allow(po.complete_moabs).to receive(:find_by!).with(moab_storage_root: ms_root).and_return(cm)
-        expect(cm).to receive(:save!)
-        expect(po).not_to receive(:save!)
-        described_class.new(druid, 1, 1, ms_root).check_existence
+        # po = create(:preserved_object, druid: druid)
```

Comment:
Should this commented-out code be removed?

Reply:
good catch, will fix!